apps/website/app/utils/supabase/apiUtils.ts

```diff
@@ -0,0 +1,156 @@
+import { NextResponse, NextRequest } from "next/server";
+import {
+  PostgrestResponse,
+  PostgrestSingleResponse,
+} from "@supabase/supabase-js";
+
+import { Database } from "@repo/database/types.gen.ts";
+import { createClient } from "~/utils/supabase/server";
+import cors from "~/utils/llm/cors";
+
+/**
+ * Sends a standardized JSON response.
+ * @param request The original NextRequest.
+ * @param data The data payload for successful responses.
+ * @param error An error message string if the operation failed.
+ * @param details Optional detailed error information.
+ * @param status The HTTP status code for the response.
+ * @param created A boolean indicating if a resource was created (influences status code: 201 vs 200).
+ */
```

Comment:
The problem with JSDoc's is that they need to be kept up to date.  If they are created at all, they should be after the review is passed and ready to be merged.  As we can see, these are already out of date.